subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -0,0 +1,1109 @@
+/*
+ * Copyright (c) 2020 Demant
```

Comment:
We can also consider if we want to add `2021` to the copyright lines

Reply:
Yes, we'll keep it in mind when making further changes